_core/smart/tutorial-counter.md

```diff
@@ -169,6 +170,8 @@ Let's get familiar with the tests to understand what the new smart contract shou
         (ok (var-get counter))))
     ```
 
+---
+
 With the completion of this tutorial, you ...
```

Comment:
I'd actually expect now to utilize my knowledge form the first tutorial and deploy and execute the contract methods. I think this should be the final step here. Otherwise, it feels like we're losing a great opportunity to connect the dots

Reply:
Added links to the deploy materials in the other tutorial